src/Sylius/Component/Core/Model/Order.php

```diff
@@ -302,12 +302,20 @@ public function hasPayment(BasePaymentInterface $payment)
      */
     public function getLastNewPayment()
     {
+        return $this->getLastCartPayment();
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getLastCartPayment()
```

Comment:
We shouldn't add this method, even better, we should get rid of `getLastNewPayment()` also if possible.